### PR TITLE
DRA-953: Accessability fixes

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Some accessability fixes here and there:

- Did a npm audit fix - removed some errors and updated the vue3-slider-component to handle some issues.
- Fixed a few issues with the slider. It now has tabindex -1 so we stick to the dropdowns if you're navigating with a keyboard.
- Reworked the checkboxes a little, to have a legend and fieldsets. A little tricky because of our structure, but my Linux screenreader is okey with this for now, as is Lighthouse. Gonna be fun to see what Monsido thinks of it.
- Found a small issue with our images and links in resultitem.
- Found a small issue in our menu - easy fix to make lighthouse happy.

### CODE REVIEW

File: src/components/search/ResultItem.vue [hunk 1]
```diff
@@ -165,7 +166,7 @@ export default defineComponent({
 		const getImageData = () => {
 			const imageDataObj = {} as ImageComponentType;
 			imageDataObj.altText = props.resultdata?.title;
-			imageDataObj.imgTitle = props.resultdata?.title;
+			imageDataObj.imgTitle = props.resultdata?.title ? props.resultdata.title : 'se materiale';
```
Comment (jorntx) [inline src/components/search/ResultItem.vue:169]: No translation needed here?

File: src/components/common/CustomTimelineSelect.vue [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
+			{{ label }}:
+			<select
+				v-model="selected"
+				@change="updated($event)"
 			>
-				{{ item }}
-			</option>
-		</select>
+				<option
+					value="1992"
```
Comment (jorntx) [inline src/components/common/CustomTimelineSelect.vue:10]: Wouldn't it be better to fetch this value from TimeSearchInitValues?

File: src/components/common/CustomTimelineSelect.vue [hunk 3]
```diff
@@ -1,26 +1,30 @@
...
+					disabled
+					selected
+					hidden
+				>
+					1992
```
Comment (jorntx) [inline src/components/common/CustomTimelineSelect.vue:15]: Wouldn't it be better to fetch this value from TimeSearchInitValues?

**Review by jorntx — COMMENTED** [review]
I'm happy with the changes and app still works but as you mentioned Monsido really has the last word here.

Only have a few comments and then we are ready to merge.

File: src/components/search/ResultItem.vue (hunk 1 above)
Comment (jesperlauridsen) [reply src/components/search/ResultItem.vue]: Done!

**Review by jorntx — APPROVED** [review]
Looks good!

**Pull request merged**